samples/todo-app/frontend/server.js

```diff
@@ -24,7 +24,7 @@ app.prepare().then(() => {
         });
     });
 
-    var bus = serviceBus.bus({ url: process.env.STATS_QUEUE_URI });
+    var bus = serviceBus.bus({ url: "amqp://"+process.env.STATS_QUEUE_SERVICE_HOST+":"+process.env.STATS_QUEUE_SERVICE_PORT_TCP });
```

Comment:
💡☕️ Might be worth to just add debug output for how and why `process.env.STATS_QUEUE_URI` to this `"amqp://"+process.env.STATS_QUEUE_SERVICE_HOST+":"+process.env.STATS_QUEUE_SERVICE_PORT_TCP` is valid.

* Also, super Nit -  we could do string literalise it like: `Foo ${object}` 

```.ts
var bus = serviceBus.bus({ url: `amqp://${process.env.STATS_QUEUE_SERVICE_HOST}:${process.env.STATS_QUEUE_SERVICE_PORT_TCP}` });
```

Reply:
Tats , like the logger statement ?

Reply:
☕️💭💡nice, re: ampq - might be needed for rabbitmq 🤷🏽‍♂️ 

☕️ Also, the nit above is just to use the goodness of  template literals (back tick) with interpolated expressions  rather than string concatenation, all your choice whatever you feel like. 

Thanks heaps 🥷🥷

Reply:
Thanks for these good suggestions. Sample value of process.env.STATS_QUEUE_URI would be ```amqp://stats-queue``` which includes the ampq, I think that's why Elena would have suggested.  @elenavillamil It didn't work with stats_queue_uri for some reason. That's why I switched to service_host. Please see below for test results..

<img width="722" alt="image" src="https://user-images.githubusercontent.com/105889062/220029462-ab505ba1-966d-46eb-8b29-b25d81277b35.png">

<img width="914" alt="image" src="https://user-images.githubusercontent.com/105889062/220032878-e00cb13d-5be8-4ccf-b1a5-3ba856c0ff62.png">

<img width="885" alt="image" src="https://user-images.githubusercontent.com/105889062/220033012-55bf06a1-ac2e-44f5-ba05-6f37a9ed3b57.png">

Reply:
@Tatsinnit I did read about interpolated template literals vs concatenating string, both seems similar. Only benefit would readability. Updated the code accordingly. Thanks for the suggestion.